app/templates/buyers/index.html

```diff
@@ -0,0 +1,60 @@
+{% extends "_base_page.html" %}
+{% block page_title %}Your account - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+    {
+        "link": "/",
+        "label": "Digital Marketplace"
+    }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+<div class="grid-row">
+  <div class="column-full">
+    {% with
+        context = current_user.email_address,
+        heading = current_user.name
+    %}
+    {% include 'toolkit/page-heading.html' %}
+    {% endwith %}
+  </div>
+  <div class="column-full dm-account-overview-options">
+    
+    <div class="framework">
+      <div class="dmspeak">
+        <h3 class="heading-xmedium">Cloud hosting, software and support</h3>
```

Comment:
Let's lose the class completely if we don't need it (also a few lines down).

Reply:
unfortunately, we can't. Ref: http://alphagov.github.io/digitalmarketplace-frontend-toolkit/shared_scss/dmspeak.html

Reply:
oh, lol, you could have just told @TheDoubleK and me that this was an existing class name then :)

(But still gahhh)

Reply:
Ha! But still, if it's a style in the toolkit it should be imported from there, not copied into the app.

Reply:
it has been imported now - no longer using copier @TheDoubleK

Reply:
`Cloud hosting, software and support` is framework-specific content and should probably come via the content loader, but you may want to do this after labs.